packages/core/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Core styles & components",
   "main": "dist/index.js",
   "module": "dist/esm/index.js",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/esm/index.d.ts",
```

Comment:
this is tricky... we might need to produce typings at __both__ locations, e.g. next to the CJS files and next to the ESM files